jaseci_core/jaseci/svc/config.py

```diff
@@ -1,9 +1,138 @@
+import os
+
 KUBERNETES_CONFIG = {"in_cluster": True, "config": None}
 
 META_CONFIG = {
-    "automation": False,
+    "automation": os.environ.get("JSORC_AUTOMATION") == "true",
     "backoff_interval": 10,
     "namespace": "default",
     "keep_alive": ["promon", "redis", "task", "mail", "elastic"],
     "kubernetes": KUBERNETES_CONFIG,
 }
+
+USER = os.environ.get("POSTGRES_USER", "postgres")
+
+DATABASE = {
+    "enabled": os.environ.get("JSORC_DB_REGEN") == "true",
+    "pod": os.environ.get("JSORC_POD_NAME", "jaseci"),
+    "host": os.environ.get("POSTGRES_HOST", "jaseci-db"),
+    "db": os.environ.get("POSTGRES_DB", "postgres"),
+    "name": USER,
+    "user": USER,
+    "password": os.environ.get("POSTGRES_PASSWORD", "lifelogifyjaseci"),
```

Comment:
Let's not have the username/pw as plain text here. Let's just not have default value.

Reply:
Same with the USER field above.